### DISCUSSION TITLE
External Authorizer HTTP support?

### DISCUSSION THREAD

**Discussion by danthegoodman1** [projectcontour/contour#4030] [Q&A]
Looking at the docs: https://projectcontour.io/docs/v1.16.0/config/api/#projectcontour.io/v1.AuthorizationServer

> AuthorizationServer configures an external server to authenticate client requests. The external server must implement the v3 Envoy external authorization GRPC protocol (https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/auth/v3/external_auth.proto).

I was wondering if there were any plans to support HTTP based authorizers so that we could us something like Ory Oathkeeper or a custom HTTP based authorizer?

**Comment by youngnick** [comment]
There is not currently, but it seems like a reasonable request. I'll try out the "create issue from discussion" button, shall I?

**Reply by youngnick** [reply]
Okay, #4031 is logged to track this.

**Reply by danthegoodman1** [reply]
Great! Super excited for this, In the meantime I will get really good with GRPC :)

**Reply by ahmed-adly-khalil** [reply]
Same here, following for sure
